jenkinsbuilder/cccp_index_reader.py

```diff
@@ -118,23 +203,41 @@ def main(indexdlocation):
             # projectify
             with open(generated_filename, 'w') as outfile:
                 yaml.dump(project, outfile)
-
             # run jenkins job builder
             myargs = ['jenkins-jobs',
                       '--ignore-cache',
                       'update',
                       ':'.join(
                           [jjb_defaults_file, generated_filename])
                       ]
-            print myargs
-            proc = subprocess.Popen(myargs,
-                                    stdout=subprocess.PIPE)
-            proc.communicate()
+            run_command(myargs)
 
+            new_projects_names.append(
+                str(project[0]["project"]["appid"]) + "-" +
+                str(project[0]["project"]["jobid"]) + "-" +
+                str(project[0]["project"]["desired_tag"])
+            )
         finally:
```

Comment:
There's no `except` clause and all `finally:` is doing is `pass`. We might as well get rid of `finally` in that case.